docs/architectural-decision-records/029-push-submissions-into-salesforce.md

```diff
@@ -0,0 +1,51 @@
+# Push Submission Data Into Salesforce
+
+In order for CMS users to be able to review submissions in Salesforce, all submission need to be in the Salesforce database. We will first get data into Salesforce by calling Salesforce APIS from app-api on every submission. Other data sync strategies may follow. 
+
+### Decision Drivers
+
+- Data needs to get into Salesforce automatically, not by manually copying data over from the web portal. 
+- It needs to be in the Salesforce review system within an hour or so
+- We would like to be able to include a link to the Salesforce record in the email we send on submission
+- We can rely on Salesforce's builtin change tracking so we don't need to send over revisions, just per-contract and per-rate data
+- Source of truth for all submission data will be the web portal, no edits will be allowed in Salesforce, only review information will be entered there. 
+
+
+## Options
+
+### Chosen: Push data into Salesforce from app-api on submission
+
+Salesforce has a few different API options for sending data into its database from outside. We can make a REST call from app-api as part of our submission handler. This will send data into Salesforce immediately at submission time, there will be no gap. On resubmission we will send over the new data. 
```

Comment:
I'd be interested to compare actual scope between option 1 and option 2 from someone on salesfroce team experienced with implementing APIs. For a web app, option 1 is considerably lower scope than option 2. I wonder thought how much is pre built into salesforce for pub sub